CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ include(DispatchAppleOptions)
 option(ENABLE_DISPATCH_INIT_CONSTRUCTOR "enable libdispatch_init as a constructor" ON)
 set(USE_LIBDISPATCH_INIT_CONSTRUCTOR ${ENABLE_DISPATCH_INIT_CONSTRUCTOR})
 
+option(BUILD_SHARED_LIBS "build shared libraries" ON)
```

Comment:
we need the shared libraries when building to include in Swift.  For me at least, autotools/libtool was defaulting to shared libs before.